content/articles/myers-briggs-personality-test-app/index.md

```diff
@@ -0,0 +1,408 @@
+published: true
+url: /building-a-personality-test-app-with-java/
+title: Building a Personality Test App with Java
+description: The goals of this tutorial is to make convincing concepts clearer and understandable enough for beginners and show the implementation of multi-dimensional array in java in a way that enables the reader to know how arrays can be used in I/O applications such as quiz app or in a question and answer app.
+This tutorial will also, show how to write java functions  for arrays, string, and floating-point manipulations, how to use StringBuilder class in java, and make a clean implementation of formatting strings in java in relation to data representation in a table.
+author: badmus-kola
+date: 2021-08-24T00:00:00-04:35
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/building-a-personality-test-app-with-java/hero.jpg
+    alt: Personality test hero image
+---
+
+### Building a Myers Briggs Personality Test App with Java
+
+If you are trying to build a personality test app or a quiz app, this article is for you. This article will use the concept of Java arrays, StringBuilder class, exception handling, and format specifiers in Java to make a personality test app.
+<!--more-->
+### Table of Content
+
+- [How the Personality Test Works](#how-the-personality-test-works)
+- [Prerequisites](#prerequisites)
+- [Introduction to Java Arrays](#introduction-to-java-arrays)
+- [Building the Personality Test App](#building-the-personality-test-app)
+- [Explaining the Methods](#explaining-the-methods)
+- [Conclusion](#Conclusion)
+- [Reference](#reference)
+
+### How the Personality Test Works
+
+According to Isabel Briggs Myers, there are sixteen types of personality identities, and there are four vital elements of categorizing people into the various sixteen personality identities. The four key elements are:
+
+- Introversion (I) or Extroversion (E)
+- Sensing (S) or Intuition (N)
+- Thinking (T) or Feeling (F)
+- Judging (J) or Perceiving (P)
+
+You can find detailed information about the various personality types [here](https://www.truity.com/page/16-personality-types-myers-briggs).
+
+To learn more about the above key elements, kindly follow this [link](https://www.verywellmind.com/the-myers-briggs-type-indicator-2795583).
+
+### Prerequisites
+
+- Java JDK 8.0 and above
+- A Java IDE
+- Test [Questions](https://pdfcoffee.com/the-myers-test-pdf-free.html) in pdf format
+
+### Introduction to Java Arrays
+
+### Definition of Arrays
+
+An array in Java is a set of elements of the same data type, either a primitive or reference type. Java can create one-dimensional arrays and multidimensional arrays. A single-dimensional array can be represented as a list of the same data type, usually defined with one square bracket. In contrast, a multidimensional array depicts the structure of a table with rows and columns.
+
+### Declaration of Arrays
+
+We declare a single-dimensional array like this.
```

Comment:
```suggestion
We declare a single-dimensional array like this:
```

Reply:
I was only enabled to commit only this change